environment.yaml

```diff
@@ -15,10 +16,10 @@ dependencies:
 - nodejs=13.13.0
 - pandas=1.0.3
 - pillow=7.1.2
-- pip=20.1
+- pip=20.1.1
```

Comment:
@englehardt this is the kind of thing that will happen when a user adds a dependency and then needs to run repin to get it included in a PR. This same thing would have happened with requirements.in / piptools, so I think it's ok.

Reply:
yes the same thing happened, and I agree that it's okay.